src/metatensor/models/cli/eval_model.py

```diff
@@ -57,8 +57,7 @@ def eval_model(model: str, structures: str, output: str = "output.xyz") -> None:
     loaded_model = load_model(model)
     structure_list = read_structures(structures)
 
-    # since the second argument is missing,
-    # this calculates all the available properties:
-    predictions = loaded_model(structure_list)
+    # this calculates all the properties that the model is capable of predicting:
```

Comment:
Can you add the `Optional` type hint to this function as well?

Reply:
Actually it will never receive None, because of the defaults in the parsers. I think it's fine this way, right?